array-methods/map.js

```diff
@@ -0,0 +1,23 @@
+const prices = [42.42, 10, 28.2234, 3.2, 5, 12];
+
+console.log('Price objects');
+console.log(
+  prices.map((element) => {
+    const newArr = [];
+    newArr.push({ price: element, salePrice: element / 2 });
+    return newArr;
+  })
```

Comment:
`map` returns an array. You don't need to create a new one and push to it. Try again.